systemvm/patches/debian/config/opt/cloud/bin/cs/CsAddress.py

```diff
@@ -659,7 +663,10 @@ def is_guest_gateway(self, bag, ip):
         gw = interface.get_gateway()
         logging.info("Interface has the following gateway ==> %s", gw)
 
-        if bag['nw_type'] == "guest" and rip == gw:
+        guest_gw = self.config.cmdline().get_guest_gw()
+        logging.info("Interface has the following gateway ==> %s", guest_gw)
+
+        if bag['nw_type'] == "guest" and (rip == gw or rip == guest_gw):
```

Comment:
This is more like a curiosity than anything else. 
What do you think about using only `return bag['nw_type'] == "guest" and (rip == gw or rip == guest_gw)` ? It does not feel that we need the conditional here.